IapExample/src/screens/Products.tsx

```diff
@@ -38,7 +38,7 @@ export const Products = () => {
 
   const handleBuyProduct = async (sku: Sku) => {
     try {
-      await requestPurchase({sku});
+      await requestPurchase({skus: [sku]});
```

Comment:
If `iOS` does not accept the array, I think it'd be good better to alert user when array is given on `iOS`.

Maybe we can throw an error when the array has passed on `iOS` only?